File: Hackerrank/30 Days of Code/day2.py
# python
# problem : https://www.hackerrank.com/challenges/30-operators/problem

#!/bin/python3

import sys

if __name__ == "__main__":
    meal_cost = float(input().strip())
    tip_percent = int(input().strip())
    tax_percent = int(input().strip())
    total_cost = meal_cost + (meal_cost*(tip_percent+tax_percent))/100.;
    print("The total meal cost is " + str(int(round(total_cost))) + " dollars.")
    

File: Hackerrank/30 Days of Code/day2.cpp
// cpp
// problem : https://www.hackerrank.com/challenges/30-operators/problem

#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    double mc, t1, t2;
    cin >> mc >> t1 >> t2;
    cout << "The total meal cost is " << roundf(mc*(1.0 + (double)t1/100.0 + (double)t2/100.0)) << " dollars.";
    return 0;
}
